commands/rautil/genachnews.js

```diff
@@ -68,13 +68,31 @@ module.exports = class GenerateAchievementNewsCommand extends Command {
         return date1 >= date2 ? d1 : d2;
       });
 
+      // Convert date to a human readable string based on the granularity
+      const [year, month, day] = json.Released.split('-').map(Number);
```

Comment:
If the value of Released is null, is .split() undefined here?

Reply:
Good catch, added a `null` check. Though realistically the date field should be filled out before the command is even used on the set.

Reply:
Agreed - though I bet some will be missed at the moment because of our tooling discrepancy with the game page's devbox and the management app (the devbox doesn't have a release date field anymore).